explainer.md

```diff
@@ -22,7 +22,52 @@ The design process of the [Web Neural Network API](https://www.w3.org/TR/webnn/)
 
 With emerging ML innovations in both software and hardware ecosystem, one of the main challenges for the web is to bridge this software and hardware development and bring together a solution that scales across hardware platforms and works with any framework for web-based machine learning experiences. We propose the WebNN API as an abstraction for neural networks in the web browsers.
 
-![WebNN architecture](content/webnn_arch.png)
```

Comment:
Delete the PNG resource or keep it?

Reply:
I removed this. We can generate a new one when settled so folks who may reference this file won't get 404.

https://github.com/webmachinelearning/webnn/pull/847/commits/99b42f5700a82f13ac0981fe03a53e3cab7da3fe